tests/cli/run/compare-coerced/outputs/source.wdl

```diff
@@ -0,0 +1,14 @@
+version 1.2
```

Comment:
We need a comment here explaining exactly what command line functionality is being tested.

Reply:
Okay, I see. So the coercion piece here is unrelated to the test. That still makes sense to me to just add a comment to the top that says "this is a representative WDL workflow to test the behavior of `run`" or similar.